src/app/common/hooks/use-bitcoin-contracts.ts

```diff
@@ -0,0 +1,170 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { JsDLCInterface } from 'dlc-wasm-wallet';
+
+import { createMoneyFromDecimal  } from '@shared/models/money.model';
+import { RouteUrls } from '@shared/route-urls';
+import { makeRpcSuccessResponse } from '@shared/rpc/rpc-methods';
+
+import { CounterpartyWalletDetails } from '@app/pages/bitcoin-contract-request/bitcoin-contract-request';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+import { useCurrentAccountNativeSegwitDetails } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+import { initialSearchParams } from '../initial-search-params';
+import { baseCurrencyAmountInQuote } from '../money/calculate-money';
+import { i18nFormatCurrency } from '../money/format-money';
+import { satToBtc } from '../money/unit-conversion';
+import { useDefaultRequestParams } from './use-default-request-search-params';
+
+function convertUint8ArrayToHexString(uint8Array: Uint8Array) {
```

Comment:
Can't you use `bytesToHex()` here and remove this function?

Reply:
You are right, bytesToHex() is good for this. Thank you!